Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -456,7 +456,47 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
         ],
         handledNodes: [argument.id]
       )
-      return .visitChildren
+    }
+    if let unexpectedAtSign = node.unexpectedBeforeAtSignToken?.onlyToken(where: { $0.tokenKind == .atSign && !$0.trailingTrivia.isEmpty }) {
+      addDiagnostic(
+        unexpectedAtSign,
+        position: node.unexpectedBeforeAtSignToken?.endPosition.advanced(by: -1),
```

Comment:
I think this only works if there is a single space after the `@`. If there are more spaces, this will show the error message before the last space. I think what you want is. `node.unexpectedBeforeAtSignToken.endPositionBeforeTrailingTrivia`.